group/list.go

```diff
@@ -66,57 +58,127 @@ func GroupList(cmd *cobra.Command, args []string) error {
 	cmd.SilenceUsage = true
 
 	groups, err := client.GetGroups(ctx, &api.GetGroupsOptions{
-		FilterHasUsers:    users,
-		FilterHasManagers: managers,
+		FilterHasUsers:    config.users,
+		FilterHasManagers: config.managers,
 	})
 	if err != nil {
-		return fmt.Errorf("Listing Group: %w", err)
+		return fmt.Errorf("listing group: %w", err)
```

Comment:
Can you keep Errors uppercased? I think we would break some parsing in other tools with this change.